src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -6,72 +6,74 @@
 from azure.cli.core.commands import cli_command
 from azure.cli.core.commands.arm import cli_generic_update_command
 from azure.cli.core.util import empty_on_404
+from azure.cli.testsdk import get_active_api_profile
 
 from ._constants import WEBHOOK_API_VERSION
 from ._format import output_format
 from ._factory import get_acr_service_client
 
-cli_command(__name__, 'acr credential show',
-            'azure.cli.command_modules.acr.credential#acr_credential_show',
-            table_transformer=output_format, exception_handler=empty_on_404)
-cli_command(__name__, 'acr credential renew',
-            'azure.cli.command_modules.acr.credential#acr_credential_renew',
-            table_transformer=output_format)
+if get_active_api_profile() == 'latest':
```

Comment:
These modules aren't supported in stack and shared.py doesn't have apiversions for all the modules in latest. Is there any link that I can refer to find out latest module api versions?